sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/WritableColumnVector.java

```diff
@@ -147,6 +147,11 @@ private void throwUnsupportedException(int requiredCapacity, Throwable cause) {
   public abstract void putShorts(int rowId, int count, short[] src, int srcIndex);
 
   /**
+   * Sets values from [rowId, rowId + count) to [src[srcIndex], src[srcIndex + count])
```

Comment:
This description is a little vague, as the input data is `byte[]`. Can we say more about this? e.g. endianness.

Reply:
I see.

Reply:
@ueshin Line 145 may make a mistake in comment `Sets values from [rowId, rowId + count) to [src + srcIndex, src + srcIndex + count)`
It should be `Sets values from [src + srcIndex, src + srcIndex + count) to [rowId, rowId + count)`
What do you think?

If so, should we update them in this PR? Or, is it better to create another PR?

Reply:
let's update them in this PR. BTW `WritableColumnVector` may be exposed to end users, so that they can build columnar batch to data source v2 columnar scan, so the document is very important.